terraform/fullnode/aws/variables.tf

```diff
@@ -120,4 +120,20 @@ variable "utility_instance_type" {
 variable "fullnode_instance_type" {
   description = "Instance type used for validator and fullnodes"
   default     = "c5.xlarge"
+}
+
+variable "enable_indexer" {
```

Comment:
Didn't realize we gave the option to run indexer on fullnode cluster. Looks like it snuck in here: https://github.com/aptos-labs/aptos-core/commit/ad6ee9573cb4941fc4bdeb8bba1189d3d1c7d27b#diff-2c02aa155c3124ea6525b43e846d6479f8eeb36ba10b5ca00b6d03e141ce67a4 when we reverted an old change 🤷 as long as it works ig

Reply:
I can remove it if not needed anymore

Reply:
yea can you also remove `indexer.tf` ? it's not being used anywhere